### PR TITLE
Adding info in Price Display Settings doc

### PR DESCRIPTION
**Pull request by dnahirna**
## Purpose of this pull request

This pull request (PR) describes adding missing info in Price Display Settings doc

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [x] Other feature set, please specify: Price Display Settings

## Affected documentation pages

https://docs.magento.com/user-guide/tax/display-settings.html

### CODE REVIEW

**Comment by leslietilling** [comment]
@dnahirna This one has similar issue as #905 did with tax settings topic and images commit included by mistake and causing conflict.

**Comment by dnahirna** [comment]
> @dnahirna This one has similar issue as #905 did with tax settings topic and images commit included by mistake and causing conflict.

Done!

File: src/tax/display-settings.md [hunk 1]
```diff
@@ -61,6 +61,18 @@ Showing prices that include and exclude tax can be confusing to the customer. To
    - `Including Tax`
    - `Including and Excluding Tax`
 
+1. For **Display Gift Wrapping Prices**, choose one of the following:
```
Comment (leslietilling) [inline src/tax/display-settings.md:64]: Need to add icon for Commerce only option/step.

```suggestion
1. {:.ee-only}For **Display Gift Wrapping Prices**, choose one of the following:
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Need to designate Commerce only features.

File: src/tax/display-settings.md (hunk 1 above)
Comment (dnahirna) [reply src/tax/display-settings.md:64]: Done!

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
running tests

**Pull request merged**